build/concourse/pipeline.yml

```diff
@@ -33,6 +35,7 @@ resources:
       paths:
         - service/github_audit/account/779799343306/apply.tfvars
         - service/github_audit/account/779799343306/backend.tfvars
+
```

Comment:
are these empty lines intentional?

Reply:
they are indeed - we've been separating out tasks/job with a space in between each so I was just making this consistent with what we've already got